src/3rd_party/apache-log4cxx-0.10.0/KEYS

```diff
@@ -50,3 +50,150 @@ CWYBgAAKCRC+FsldLhFDIjnoAJ9ECOIrTH3adnVLOkHZnewyp2ssxwCgtLjlwZ7/
 4QtL3W5Id3nKxPFiI+c=
 =9ut1
 -----END PGP PUBLIC KEY BLOCK-----
+
```

Comment:
@SNiukalov what is this changes about? What is this keys about?

Reply:
This keys - PGP signatures for verify the integrity of the downloaded files.
They delivered with library.